addons/jest/src/register.tsx

```diff
@@ -2,15 +2,13 @@ import React from 'react';
 import addons from '@storybook/addons';
 import { ADDON_ID, PANEL_ID } from './shared';
 
-// import PanelTitle from './components/PanelTitle';
 import Panel from './components/Panel';
 
 addons.register(ADDON_ID, api => {
   const channel = addons.getChannel();
```

Comment:
We should have no need for the channel anymore, calling api.on() === channel.on()

All relavent channel APIs are available on api

Reply:
OK